<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProfile</class>
 <widget class="QWidget" name="AddProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtSixA - Add Profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_loc">
       <property name="text">
        <string>FDI file location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_loc">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="b_loc">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&lt;i&gt;(Use the &quot;Create New...&quot; button to make one)&lt;/i&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_png">
       <property name="text">
        <string>PNG file location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_png">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="b_png">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;i&gt;(for previewing; use &lt;a href=file:///usr/share/qtsixa/pics/sixa_x.xcf&gt;/usr/share/qtsixa/pics/sixa_x.xcf&lt;/a&gt; as template; then resize to half size)&lt;/i&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_short">
       <property name="text">
        <string>Short profile name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_short">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;i&gt;(non capitalized and no spaces!)&lt;/i&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_full">
       <property name="text">
        <string>Full profile name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_full">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&lt;i&gt;(If possible, use only ASCII characters; space allowed)&lt;/i&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../usr/share/qtsixa/icons/cancel.png</normaloff>../../../../../../../usr/share/qtsixa/icons/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_add">
       <property name="text">
        <string>Add Profile Now!</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../usr/share/qtsixa/icons/plus.png</normaloff>../../../../../../../usr/share/qtsixa/icons/plus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddProfile</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
